python/graphstorm/config/argument.py

```diff
@@ -187,6 +189,22 @@ def override_arguments(self, cmd_args):
                 setattr(self, f"_{arg_key}", arg_val)
                 print(f"Overriding Argument: {arg_key}")
 
+    def handle_argument_conflicts(self):
+        """Check and resolve argument conflicts
+        """
+        # 1. language model conflicts
+        if self.node_lm_configs is not None:
+            # gradient checkpoint does not work with freeze_lm_encoder_epochs
+            # When freeze_lm_encoder_epochs is set, turn off gradient checkpoint
```

Comment:
if gradient checkpoint is enabled, we cannot freeze the model training of the BERT models?